### PR TITLE
Pass all `<Html>` props to `<html>`

### PR DESCRIPTION
**Pull request by Timer**
Follow-up to #6372

### CODE REVIEW

File: packages/next/pages/_document.js [hunk 1]
```diff
@@ -50,7 +50,11 @@ export class Html extends Component {
 
   render() {
     const { amphtml } = this.context._documentProps
-    return <html amp={amphtml ? '' : null}>{this.props.children}</html>
+    return (
+      <html amp={amphtml ? '' : null} {...this.props}>
```
Comment (giuseppeg) [inline packages/next/pages/_document.js]: maybe for consistency the `amp` attribute should always come from context and override `props.amp`. Probably the amp validator will complain about this but you could spare that to the end user

Comment (Timer) [reply packages/next/pages/_document.js]: Yeah, I guess if `amphtml` is set we are rendering as AMP -- nothing the user can do to opt out.

**Review by dav-is — APPROVED** [review]

File: packages/next/pages/_document.js [hunk 2]
```diff
@@ -50,7 +50,12 @@ export class Html extends Component {
 
   render() {
     const { amphtml } = this.context._documentProps
-    return <html amp={amphtml ? '' : null}>{this.props.children}</html>
+    const { children, ...props } = this.props
```
Comment (giuseppeg) [inline packages/next/pages/_document.js]: nit: you can avoid this if you use a self closing tag:

```jsx
return <html {...this.props} amp={amphtml ? '' : null} />
```

Comment (timneutkens) [reply packages/next/pages/_document.js]: Let's do that in another PR 👌

**Pull request merged**